packages/connect/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 |             Package              | Stable |    Canary    |
 | :------------------------------: | :----: | :----------: |
-|       npm @trezor/connect        | 9.2.4  | 9.2.5-beta.1 |
-|     npm @trezor/connect-web      | 9.2.4  | 9.2.5-beta.1 |
-| npm @trezor/connect-webextension | 9.2.4  | 9.2.5-beta.1 |
+|       npm @trezor/connect        | 9.2.4  | 9.2.5-beta.2 |
+|     npm @trezor/connect-web      | 9.2.4  | 9.2.5-beta.2 |
+| npm @trezor/connect-webextension | 9.2.4  | 9.2.5-beta.2 |
 
-|     Deployment     | Stable |    Canary    |
-| :----------------: | :----: | :----------: |
-| connect.trezor.io/ | 9.2.4  | 9.2.5-beta.1 |
+|     Deployment      | Stable |    Canary    |
+| :-----------------: | :----: | :----------: |
+| connect.trezor.io/9 | 9.2.4  | 9.2.5-beta.2 |
```

Comment:
why this change `/9` ?

Reply:
Because it was previously changed manually by this PR https://github.com/trezor/trezor-suite/pull/12959

But the script that generates the table automatically was not updated.

@Hannsek Shall I update the script so that is always `connect.trezor.io/` ?

Reply:
Aha, didn't know that. 

Then yes, please do.

Reply:
@Hannsek  :ok: Done https://github.com/trezor/trezor-suite/pull/13077